network-api/networkapi/templates/fragments/buyersguide/pni_category_nav.html

```diff
@@ -1,14 +1,10 @@
 {% load bg_nav_tags localization i18n static wagtailroutablepage_tags wagtailmetadata_tags %}
-<nav id="multipage-nav" class="pni-category-nav text-center d-none d-md-block tw-no-scrollbar tw-mt-2" title="{% trans "site navigation" context "Tooltip on menu items" %}">
+<nav id="multipage-nav" class="pni-category-nav text-center d-none d-md-block tw-no-scrollbar" title="{% trans "site navigation" context "Tooltip on menu items" %}">
     <div class="container tw-py-2" id="product-review">
       <div class="row">
         <div class="col">
           <div id="pni-category-wrapper" class="tw-flex tw-items-center tw-w-max tw-min-w-full">
-            {% if pagetype == "product" or pagetype == "about" %}
-              <a class="multipage-link tw-block active" data-name="None" data-mobile="{% trans "All Categories" %}" href="{% relocalized_url home_page.localized.url %}">{% trans "All Product Reviews" %}</a>
-            {% else %}
-              <a class="multipage-link tw-block {% if not category %} active{% endif %}" data-name="None" data-mobile="{% trans "All Categories" %}" href="{% relocalized_url home_page.localized.url %}">{% trans "All Reviews" %}</a>
-            {% endif %}
+            <a class="multipage-link tw-block {% if pagetype == "home" %} active {% endif %}" data-name="None" data-mobile="{% trans "All Categories" %}" href="{% relocalized_url home_page.localized.url %}">{% trans "All Reviews" %}</a>
```

Comment:
Can you add some context why you decided to keep using a page type check here?

Reply:
@tbrlpld  We want `All Reviews` to be in a starting active state only on the homepage, this change is to reflect the new pages that the new category nav included. Similar setup to setting up the initial active state on the category nav

Reply:
Sorry, I should have been clearer in that question. I understood why we want to be able to only show the "active" state on pages that contain the product grid. 

My question was more around why use a check like "if page type is something" instead of "if show active category"?